src/Facebook/InstantArticles/Client/Helper.php

```diff
@@ -37,6 +37,7 @@ public function __construct($facebook)
      * @param string $appID
      * @param string $appSecret
      *
+     * @return static
```

Comment:
@everton-rosario 

Actually, @mvriel (maintainer of https://www.phpdoc.org) says no, see https://github.com/php-fig/http-message/pull/50#issuecomment-152563990.